Responsive Grid and Off Canvas Menu

asommer70 opened the pull request:
Hi,

This a pretty big pull request, so if you'd like me to separate out some of the changes I can totally do that.  I made some pretty significant changes to the layout and some of the pages:

* Integrate [Susy](http://susy.oddbird.net/) for building a grid in SASS.
* Set the Electron window to be able to be resized.
* Moved the side bar into an off-canvas menu.
* Moved some of the sidebar methods into a menu above the Articles list.
* Moved the Manage Feeds button to the off-canvas menu.
* Integrated the Add Feed functionality into the Manage Feeds view.
* Removed Add Feed button from the off-canvas menu because it seems to fit better in the Mange Feeds page.

Again, if this is too much, or you'd like me to tidy up anything please let me know.

Thanks,
Adam

mrgodhani commented:
@asommer70 Yes if you can please separate out pull request that would be great. It would be easy for me to review it.

asommer70 commented:
Created two additional pull requests with the past two commits that introduced Susy and the header bar.  This commit is the final one with the off-canvas sidebar integrated.

Please let me know if this is what you need to better review the code.

Thanks @mrgodhani

asommer70 commented:
So there's now some additional code in my fork.  I added a "Import OPML" functionality to the Manage Feeds page and fixed the sidebar menu to close once something is selected.  

So the sidebar works a lot better now.

I know it's a lot of code changes so no rush.

Thanks @mrgodhani
